### DISCUSSION TITLE
Building HPX

### DISCUSSION THREAD

**Discussion by Hassnaa9** [STEllAR-GROUP/hpx#6460] [General]
## Behavior
while building hpx i get some errors and try to fix it so i found an easy way to build the source code directly without facing a lot of errors

## Steps to Reproduce the Problem

to build hpx using linux
run this commands in your terminal

1-in the directory which you install hpx source code
mkdir build && cd build

2-make install folder in a place you know well

3-run the following commands in terminal :
cmake -DBOOST_ROOT=path/boost_1_82_0 
-DHWLOC_ROOT=path/hwloc-2.10.0 
-DASIO_ROOT=path/asio-1.28.0 
-DCMAKE_INSTALL_PREFIX=/the path of/ install dir
-DHPX_WITH_MALLOC=system ..

4-
cmake --build . --target install

... Please describe your environment 
Linux Ubuntu

**Comment by hkaiser** [comment]
Was this information not sufficient for you to get started: https://hpx-docs.stellar-group.org/branches/master/html/manual/building_hpx.html#unix-variants?
